### DISCUSSION TITLE
Admin User on Premise

### DISCUSSION THREAD

**Discussion by ReneGraetz** [outline/outline#2096] [General] [answered]
Hi,

everything is up and running but i wonder how i become an admin?
The first user who has logged in over microsoft is not the admin.
where can i change user to administrators?

thanks

**Comment by tommoor** [comment, marked as answer]
The first user created for a team is automatically the admin. If this is somehow not the case they only way to resolve would be to flip the `isAdmin` boolean in the database directly

**Reply by ReneGraetz** [reply]
Ok where can i find this?

**Reply by ReneGraetz** [reply]
Worked like a charm

You can find it here:

![image](https://user-images.githubusercontent.com/78431484/116534423-c9675600-a8e2-11eb-9ca5-e53a95652a9d.png)

**Reply by xargi** [reply]
I ran into this problem on my first try when the first login failed due to a configuration issue, which I think was too short secret key. The first user created after fixing that was not an admin. After resetting database, it worked on the first try.
